### DISCUSSION TITLE
Has anyone tested Intel 2732 ?

### DISCUSSION THREAD

**Discussion by ABelliqueux** [TomNisbet/TommyPROM#76] [Q&A]
In the doc, I found that Intel 2732s might be supported by `PromDevice27` and a 25V power supply. Has anyone tested this ?

**Comment by TomNisbet** [comment]
I have some intel B2732 chips that I haven't tested yet.  Pay careful attention to the data sheets for 27 series chips because the specs can vary between different manufacturers or even different part numbers for the same manufacturer.  The intel 2732 sheet lists Vpp as 25V but the 2732A specifies Vpp=21V and states that anything over 22V will damage the chip.

What is the part number on your chip?  If I get a chance today, I'll try to program one of mine.

**Reply by ABelliqueux** [reply]
Hi, 
Thanks for your reply ! I don't have the part with me right now, but I might get a chance to look at it soon. I'll let you know if I can get more details.

**Reply by ABelliqueux** [reply]
Hi @TomNisbet, I still don't have the exact part number, but did you get a chance to try with one of yours ?
